.github/workflows/release-stable.yml

```diff
@@ -54,33 +26,55 @@ jobs:
           distribution: 'zulu'
           java-package: 'jdk+fx'
 
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
-
       - name: Gradle build
         uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
         with:
           arguments: build
 
+      - name: Remove Snapshot Suffix
+        id: removeSnap
+        uses: gradle/gradle-build-action@842c587ad8aa4c68eeba24c396e15af4c2e9f30a
+        with:
+          arguments: removeSnapshotSuffix
+
+      - name: Remove Snapshot Suffix
+        run: |
+          git config --global user.name 'acrolinx-machine-user-for-integrations'
+          git config --global user.email 'danjela.lura@acrolinx.net'
```

Comment:
We should not hard-code Danjela's email and use that for every release commit

Reply:
This will be used for testing purposes only, it's not final yet since we need to create a service account that can be used to make the intermediate commits.